tap_shopify/__init__.py

```diff
@@ -158,27 +176,30 @@ def sync():
         LOGGER.info('%s: %d', stream_id, stream_count)
     LOGGER.info('----------------------')
 
-@utils.handle_top_exception(LOGGER)
```

Comment:
This doesn't feel like the right move. The decorator in `singer-python` should be updated to do the splitlines here: https://github.com/singer-io/singer-python/blob/master/singer/utils.py#L222

Reply:
Updated singer-python and reverted this change.